src/main/java/mate/academy/service/AuthenticationService.java

```diff
@@ -11,6 +13,11 @@ public class AuthenticationService {
      * Return false in any other cases.
      */
     public boolean login(String email, String password) {
-        return false;
+        UserService userService = new UserService();
```

Comment:
should be class-level `private final` field

Reply:
I have just added new commit on by branch.

Reply:
> should be class-level `private final` field

I have just added new commit on by branch, but can't see if it's checked by Java CI.